content/blog/integrating-istio-and-spire.md

```diff
@@ -0,0 +1,463 @@
+pod/details-v1-f8957ccb4-7vdgw       2/2     Running   0             37d
+pod/productpage-v1-cfb4bc854-5km2l   2/2     Running   0             37d
+pod/ratings-v1-65cd6fbcd8-s9jnc      2/2     Running   0             37d
+pod/reviews-v1-55f769fb78-czh7j      2/2     Running   0             37d
+pod/reviews-v2-6b7c798cc8-wkpxg      2/2     Running   0             37d
+pod/reviews-v3-695c7f59db-nzwwk      2/2     Running   2 (34d ago)   37d
+
+NAME                  TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+service/details       ClusterIP   10.111.38.161    <none>        9080/TCP   37d
+service/productpage   ClusterIP   10.102.189.161   <none>        9080/TCP   37d
+service/ratings       ClusterIP   10.105.7.153     <none>        9080/TCP   37d
+service/reviews       ClusterIP   10.106.49.246    <none>        9080/TCP   37d
+
+NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/details-v1       1/1     1            1           37d
+deployment.apps/productpage-v1   1/1     1            1           37d
+deployment.apps/ratings-v1       1/1     1            1           37d
+deployment.apps/reviews-v1       1/1     1            1           37d
+deployment.apps/reviews-v2       1/1     1            1           37d
+deployment.apps/reviews-v3       1/1     1            1           37d
+
+NAME                                       DESIRED   CURRENT   READY   AGE
+replicaset.apps/details-v1-f8957ccb4       1         1         1       37d
+replicaset.apps/productpage-v1-cfb4bc854   1         1         1       37d
+replicaset.apps/ratings-v1-65cd6fbcd8      1         1         1       37d
+replicaset.apps/reviews-v1-55f769fb78      1         1         1       37d
+replicaset.apps/reviews-v2-6b7c798cc8      1         1         1       37d
+replicaset.apps/reviews-v3-695c7f59db      1         1         1       37d
+```
+
+Once everything is up, all workloads would get registered under SPIRE server.
+
+**4.6** You can verify the registration of workloads using the following command:
+
+```shellsession
+kubectl exec <spire-server_pod_name> -n spire -c spire-server -- ./bin/spire-server entry show
+```
+
+Verify that every workload with same label as clusterSPIFFEID CRD’s match label is registered in the server.
+
+![](/img/server-entries.png)
+
+**4.7** Verify that the certificate issuer of workloads is SPIRE using following commands for each workload.
+
+```shellsession
+istioctl proxy-config secret <pod_name> -n <namespace_name> -o json | jq -r '.dynamicActiveSecrets[0].secret.tlsCertificate.certificateChain.inlineBytes' | base64 --decode > chain.pem
+```
+
+```shellsession
+k8s-spiffe-integ-master-7j7fh-m67q9:~ openssl x509 -in chain.pem -text | grep SPIRE
+ Subject: C = US, O = SPIRE, x500UniqueIdentifier = e2f9c35b9198e1824373e874b13287d0
+```
+
+You should also check the same for ingress-gateway pod in Istio-system namespace and verify that your deployed workloads and ingress-gateway has the same issuer.
```

Comment:
Recommend rephrasing the beginning of this sentence to:
You should also check that everything is the same for the ingress-gateway pod...